deploy-disconnected-registry/ocp-sync.sh

```diff
@@ -56,7 +58,7 @@ function mirror_ocp() {
     while [ ${retry} != 0 ]; do
         # Mirror ocp release with retry strategy
         echo DEBUG: "oc --kubeconfig=${TARGET_KUBECONFIG} adm release mirror -a ${PULL_SECRET} --from=${OPENSHIFT_RELEASE_IMAGE} --to-release-image=${OCP_DESTINATION_INDEX} --to=${DESTINATION_REGISTRY}/${OCP_DESTINATION_REGISTRY_IMAGE_NS}"
-        oc --kubeconfig=${TARGET_KUBECONFIG} adm release mirror -a ${PULL_SECRET} --from="${OPENSHIFT_RELEASE_IMAGE}" --to-release-image="${OCP_DESTINATION_INDEX}" --to="${DESTINATION_REGISTRY}/${OCP_DESTINATION_REGISTRY_IMAGE_NS}" >>${OUTPUTDIR}/mirror-ocp.log 2>&1
+        oc --kubeconfig=${TARGET_KUBECONFIG} adm release mirror -a ${PULL_SECRET} --from="${OPENSHIFT_RELEASE_IMAGE}" --to-release-image="${OCP_DESTINATION_INDEX}" --to="${DESTINATION_REGISTRY}/${OCP_DESTINATION_REGISTRY_IMAGE_NS}" >>${OUTPUTDIR}/mirror-ocp.log
```

Comment:
please add the redirection because in the pipeline we decided reduce the information for the user...(I understand you did just to verify during dev step, but to be merged in main we should remove it)